namespaces/nsenter/nsenter.go

```diff
@@ -5,6 +5,7 @@ package nsenter
 /*
 __attribute__((constructor)) init() {
 	nsenter();
```

Comment:
do we need to call `nsenter` here with the new implementation of `ExecIn` ? I think `nsexec` alone would suffice right, as `nsenter` will just exit right away because `argv[0]` does not start with `nsenter-` anymore.